hdfs-scheduler/src/main/java/org/apache/mesos/hdfs/scheduler/HdfsNode.java

```diff
@@ -109,11 +116,14 @@ private ExecutorInfo createExecutor(String taskIdName, String nodeName, String n
       CommandInfoBuilder.createCmdInfoUri(config.getJreUrl()));
   }
 
-  private List<Environment.Variable> getExecutorEnvironment() {
-    return Arrays.asList(
-      EnvironmentBuilder.createEnvironment("LD_LIBRARY_PATH", config.getLdLibraryPath()),
-      EnvironmentBuilder.createEnvironment("EXECUTOR_OPTS", "-Xmx" + config.getExecutorHeap() + "m -Xms" +
-        config.getExecutorHeap() + "m"));
+  protected List<Environment.Variable> getExecutorEnvironment() {
+    List<Environment.Variable> env = EnvironmentBuilder.
```

Comment:
We are using "MESOS" multiple times in multiple classes. Let's add it to HdfsConstants